spec/vast_trackers.spec.js

```diff
@@ -13,13 +13,15 @@ describe('VASTTracker', function() {
     let spyTrack;
     let adTrackingUrls;
     let ad;
+    let spyAdError;
+    let isCustomCode;
```

Comment:
As there is only few tests that use the variable `isCustomCode` maybe we could define it there instead of "globally"